docs/02-app/02-api-reference/08-next-cli.mdx

```diff
@@ -175,3 +175,6 @@ will give you information like this example:
 ```
 
 This information should then be pasted into GitHub Issues.
+
+In case of diagnosing installation issue in more detail, you can run `next info --verbose` to get the additional information about system and the installation of next-related packages.
+This will generate lengthy output, if it's hard to copy paste running `next info --verbose --out-file filename` will generate text report file instead of emitting to stdout.
```

Comment:
```suggestion
In order to diagnose installation issues, you can run `next info --verbose` to print additional information about system and the installation of next-related packages. This output could be lengthy so you can also run `next info --verbose --out-file file.txt` to write a text report file instead of emitting to stdout.
```